src/Letter.ts

```diff
@@ -0,0 +1,119 @@
+import type { TDocumentDefinitions, ContentText, ContentImage, Content } from 'pdfmake/interfaces';
+import { layout as dinLayoutFunction } from './layouts/din-5008-a';
+import { formatAddress, mm2pt, stripTags, applyFunctionToPdfMakeContent } from './utility';
+import type { LayoutFunction, PdfMakeContent, Signature } from './types';
+
+type LetterProps = {
+    sender_address: string[] | string;
+    recipient_address: string[] | string;
+    /** The information block usually displayed at the top-right of the letter. */
+    information_block?: PdfMakeContent;
+    subject: string;
+    content: PdfMakeContent;
+    signature: Signature;
+};
+// The `marginTop` is missing in the type definitions.
+type PdfMakeSignature = (ContentText | ContentImage) & { marginTop: number };
+
+const _parseTags = (content: string) => {
```

Comment:
This could use a return type.

Reply:
Anything wrong with the inferred type? Looks fine to me.

Reply:
I like the code to expressive acutally.